cmd/tidb-dashboard/main.go

```diff
@@ -228,3 +184,62 @@ func main() {
 	wg.Wait()
 	log.Info("Stop dashboard server")
 }
+
+func init() {
+	cfg.CoreConfig = &config.Config{}
+	rootCmd.Flags().BoolVarP(&showVersion, "version", "v", false, "Print version information and exit")
```

Comment:
maybe we can utilize cobra's own version functionality https://github.com/spf13/cobra#version-flag and use `version` command.

Reply:
got it